minvws/nl-kat-coordination
# Fix KAT-alogus navigation

### Changes
Fixing the navigation of the KAT-alogus. There was a bug when a user went to the KAT-alogus and then immediately clicked on the list-view. The page then redirected to the "All" tab.

- Changed the URL of "All" from `/kat-alogus/` to `/kat-alogus/plugins/all/grid/`
- Added class KATalogusLandingView to redirect to the Boefje tab after clicking on "KAT-alogus" in the navigation bar.

### QA notes
Please do the following:
- Go to "KAT-alogus"
- Check if the URL is `/kat-alogus/plugins/boefjes/grid/`
- Click on the list-view
- Check if the URL is `/kat-alogus/plugins/all/table/`

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
